src/sugar3/graphics/objectchooser.py

```diff
@@ -42,16 +42,19 @@
 
 
 def get_preview_pixbuf(preview_data, width=-1, height=-1):
-    """Retrive a pixbuf with the content of the preview field
+    """
+    Retrive a pixbuf with the content of the preview field
 
-    Keyword arguments:
-    metadata -- the metadata dictionary.
-                Can't be None, use metadata.get('preview', '')
-    width -- the pixbuf width, if is not set, the default width will be used
-    height -- the pixbuf width, if is not set, the default height will be used
+    Args:
+        metadata (dictionary): the metadata dictionary. Can't be None, use metadata.get('preview', '')
```

Comment:
Maybe put a usage example that uses the `metadata.get('preview;, '')` after the summary (2 lines above)